lib/plugin/wdio.js

```diff
@@ -99,7 +99,8 @@ module.exports = (config) => {
     if (Service) {
       if (Service.launcher && typeof Service.launcher === 'function') {
         const Launcher = Service.launcher;
-        launchers.push(new Launcher(config));
+        const options = { logPath: global.output_dir, installArgs: {}, args: {} };
+        launchers.push(new Launcher(options, [config.capabilities], config));
```

Comment:
is this backwards compatible with webdriverio 5?

Reply:
With that code, no. 😅. Maybe need to add the condition to make it backwards compatible with v5.

Reply:
@DavertMik @pablopaul made it be compatible with v5.